packages/utils/src/worldwide.ts

```diff
@@ -44,34 +68,23 @@ export interface InternalGlobal {
    */
   _sentryDebugIds?: Record<string, string>;
   __SENTRY__: {
-    hub: any;
-    logger: any;
-    extensions?: {
-      /** Extension methods for the hub, which are bound to the current Hub instance */
-      // eslint-disable-next-line @typescript-eslint/ban-types
-      [key: string]: Function;
-    };
-    globalScope: Scope | undefined;
-    defaultCurrentScope: Scope | undefined;
-    defaultIsolationScope: Scope | undefined;
-    globalMetricsAggregators: WeakMap<Client, MetricsAggregator> | undefined;
-    /** Overwrites TextEncoder used in `@sentry/utils`, need for `react-native@0.73` and older */
-    encodePolyfill?: (input: string) => Uint8Array;
-    /** Overwrites TextDecoder used in `@sentry/utils`, need for `react-native@0.73` and older */
-    decodePolyfill?: (input: Uint8Array) => string;
-  };
+    [key: VersionString]: BackwardsCompatibleSentryCarrier;
+    version: VersionString;
+  } & BackwardsCompatibleSentryCarrier;
   /**
    * Raw module metadata that is injected by bundler plugins.
    *
    * Keys are `error.stack` strings, values are the metadata.
    */
   _sentryModuleMetadata?: Record<string, any>;
   _sentryEsmLoaderHookRegistered?: boolean;
-}
+};
 
 /** Get's the global object for the current JavaScript runtime */
 export const GLOBAL_OBJ = globalThis as unknown as InternalGlobal;
 
+type SingletonKeys = Exclude<keyof InternalGlobal['__SENTRY__'], 'version' | VersionString>;
```

Comment:
If we split up into `SentryCarrier` & `BackwardsCompatibleSentryCarrier` as proposed above, then could we not just do `keyof SentryCarrier` here? 🤔

Reply:
Now that I think about it, I'm not even sure if we need `BackwardsCompatibleSentryCarrier` at all 🤔 any chance we just never cleaned this type up properly? I checked and we only use this type within utils and once in SvelteKit (which we can get rid of somehow, I'm fairly sure). It's exported from utils but we don't guarantee semver for this package. we could just remove the unused properties.

Reply:
ok screw this, ReactNative needs some properties on `InternalGlobal`. I'm still gonna pull `SentryCarrier` & `BackwardsCompatibleSentryCarrier` apart but will leave all props in place for now

Reply:
done in [efe3229](https://github.com/getsentry/sentry-javascript/pull/12206/commits/efe3229268447c4a0e26d1cdefdb1b3f965ca71a)